plugin/api.go

```diff
@@ -65,6 +65,9 @@ type API interface {
 	// The status parameter can be: "online", "away", "dnd", or "offline".
 	UpdateUserStatus(userId, status string) (*model.Status, *model.AppError)
 
+	// GetUsersInChannelByStatus returns a page of users in a channel. Page counting starts at 0. Sorted by Status.
+	GetUsersInChannelByStatus(channelId string, page, perPage int) ([]*model.User, *model.AppError)
```

Comment:
@jwilander, notwithstanding the `app` method of the same name, I find it unfortunate to introduce a "new" `GetXByY` method that doesn't have the behaviour of getting `X` filtering by `Y` (e.g. `GetUsersByIds`, `GetUsersByUsernames`)

Could we break the `Client4` API here and go with something like `GetUsersInChannelSortByStatus` or maybe a `GetSortedUsersInChannel` that takes a `sort` parameter like `GetSortedEmojiList`?

Reply:
I think that makes sense, 0/5 on which way to go

Reply:
👍 Yep, that sounds reasonable!